File: backend-api/app/Models/Organization.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Organization extends Model
{
    use HasFactory;

    protected $fillable = ['name', 'description', 'address_id'];

    protected $hidden = ['address_id', 'created_at', 'updated_at'];

    public function address()
    {
        return $this->belongsTo(Address::class);
    }
    public function users()
    {
        return $this->belongsToMany(User::class, 'organization_user');
    }
}

File: frontend-desktop-app/SmartFridge/Models/Organization.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SmartFridge.Models
{
    public class Organization
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public Address Address { get; set; }
    }
}
